idd/Energy+.idd.in

```diff
@@ -19458,6 +19458,7 @@ RoomAir:Node:AirflowNetwork:InternalGains,
       \key ElectricLoadCenter:Inverter:LookUpTable
       \key ElectricLoadCenter:Storage:Battery
       \key ElectricLoadCenter:Storage:Simple
+      \key ElectricLoadCenter:Storage:Converter
```

Comment:
This addition is ok, it would be a new feature that openstudio has not yet wrapped.  The issue would be on translation from idd to osm, we need to put this in our e+ idd or else we could not open idf files with this field.  So it is ok but we do need to know about it.  Is this going in?